src/client/Microsoft.Identity.Client/ApiConfig/AcquireTokenByAuthorizationCodeParameterBuilder.cs

```diff
@@ -88,7 +88,8 @@ internal override Task<AuthenticationResult> ExecuteInternalAsync(CancellationTo
         public AcquireTokenByAuthorizationCodeParameterBuilder WithSendX5C(bool withSendX5C)
         {
             CommonParameters.AddApiTelemetryFeature(ApiTelemetryFeature.WithSendX5C);
-            Parameters.SendX5C = withSendX5C;
+            Parameters.SendX5C = withSendX5C; 
+            Parameters.SetPerRequestX5C = true;
```

Comment:
Can we move `sendX5C` to AbstractConfidentialClientAcquireTokenParameterBuilder instead of having this repeat code?